addons/im_livechat/static/src/embed/common/boot_helpers.js

```diff
@@ -39,6 +39,7 @@ async function loadFont(name, url) {
 export function makeRoot(target) {
     const root = document.createElement("div");
     root.classList.add("o-livechat-root");
+    root.setAttribute("id", "liveChatRoot");
```

Comment:
I would name it `o-livechat-root` like the classname. `o-` prefix for "odoo" reduces risks of conflicts with non-Discuss/Livechat code.

Reply:
I'm aware of very few customers that use more than a single livechat on their websites, the hack being to use `website_livechat` and external lib livechat simultaneously.

Considering this `root` is available in the light DOM, would be nice to allow stacking more than a single livechat with parts of the id being unique. Something like: `luxon.DateTime.now().ts + Math.random()`